src/evmTest/scala/io/iohk/ethereum/vm/ThrowSpec.scala

```diff
@@ -13,7 +13,7 @@ class ThrowSpec extends FreeSpec with Matchers {
 
       val result = contract.justThrow().call()
 
-      result.error shouldBe Some(InvalidOpCode) // TODO: change to invalid jump destination
+      result.error shouldBe Some(InvalidOpCode(0xfd.toByte))
```

Comment:
well I have just run them and getting the opposite result:
`Some(InvalidOpCode(0xfd)) was not equal to Some(InvalidJump) (MinimumViableTokenSpec.scala:34)`